coda-automation/services/prometheus/prometheus.j2

```diff
@@ -0,0 +1,141 @@
+# my global config
```

Comment:
see below re: CRUFT + obsolete infra (may be sensible to reuse some of this now tho curious what others think)